src/collects/seashell/backend/files.rkt

```diff
@@ -136,18 +136,24 @@
   (void))
 
 ;; (read-file project file) -> bytes?
-;; Reads a file as a Racket bytestring.
+;; Reads a file and its undoHistory as Racket bytestrings.
 ;;
 ;; Arguments:
 ;;  project - project to read file from.
 ;;  file - name of file to read.
 ;;
 ;; Returns:
-;;  Contents of the file as a bytestring.
+;;   (list contents undoHistory)
 (define/contract (read-file project file)
```

Comment:
Return a (values contents UndoHistory) instead of a list of two elements.
It's more consistent with the rest of the codebase and most Racket
libraries.
On May 25, 2016 11:17, "mbardakov" <notifications@github.com> wrote:

> In src/collects/seashell/backend/files.rkt
> <https://github.com/cs136/seashell/pull/547#discussion_r64591706>:
>
> >  ;;
> >  ;; Arguments:
> >  ;;  project - project to read file from.
> >  ;;  file - name of file to read.
> >  ;;
> >  ;; Returns:
> > -;;  Contents of the file as a bytestring.
> > +;;   (list contents undoHistory)
> >  (define/contract (read-file project file)
>
> Not sure what you mean here?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cs136/seashell/pull/547/files/69d72931f2a16e29be6f9e84964701507560eee5#r64591706>
>